packages/ra-core/src/controller/input/ReferenceArrayInputController.spec.tsx

```diff
@@ -198,34 +205,15 @@ describe('<ReferenceArrayInputController />', () => {
                 {children}
             </ReferenceArrayInputController>
         );
-        assert.equal(children.mock.calls[0][0].warning, undefined);
-    });
-
-    it('should pass onChange down to child component', () => {
```

Comment:
why did you remove this one?

Reply:
Because onChange override is handled at the input level. See the `useInput` hook